src/ports/postgres/modules/cols_vec/cols2vec.py_in

```diff
@@ -0,0 +1,110 @@
+"""
+@file cols2vec.py_in
+
+@brief Utility to convert Columns to array
+
+"""
+
+import plpy
+from utilities.control import MinWarning
+from utilities.utilities import split_quoted_delimited_str
+from utilities.utilities import _string_to_array
+from utilities.validate_args import columns_exist_in_table
+from utilities.validate_args import is_var_valid
+from utilities.validate_args import get_cols
+from utilities.validate_args import quote_ident
+from utilities.utilities import py_list_to_sql_string
+
+
+m4_changequote(`<!', `!>')
+
+
+def validate_cols2vec_args(source_table, output_table,
+                           list_of_features, list_of_features_to_exclude, cols_to_output, **kwargs):
+    """
+        Function to validate input parameters
+    """
+    if list_of_features.strip() != '*':
```

Comment:
The checks below are better expressed as assert statements using `_assert(...)`

Reply:
This is done.